doc/COMPILE.md

```diff
@@ -606,6 +606,27 @@ running cmake like to enable debug symbol.
 For Gentoo it's recommended to emerge sys-apps/lsb-release to name the package
 right even thou not required.
 
+### With Flatpak (Beta)
```

Comment:
Better put that in a README.md in scripts/flatpak

Reply:
Done